controllers/storagecluster/provider_server.go

```diff
@@ -96,7 +93,7 @@ func (o *ocsProviderServer) ensureDeleted(r *StorageClusterReconciler, instance
 	return reconcile.Result{}, finalErr
 }
 
-func (o *ocsProviderServer) createDeployment(r *StorageClusterReconciler, instance *ocsv1.StorageCluster) error {
+func (o *ocsProviderServer) createDeployment(r *StorageClusterReconciler, instance *ocsv1.StorageCluster) (reconcile.Result, error) {
```

Comment:
No need to introduce a result into the sig of this func. Please revert

Reply:
I changed this one as I am returning a requeue in this one as well when the server pod is not ready.

Reply:
ok